src/client/src/tscParseLineProtocol.c

```diff
@@ -843,7 +1109,7 @@ static int32_t doInsertChildTableWithStmt(TAOS* taos, char* sql, char* cTableNam
       tscError("SML:0x%"PRIx64" taos_stmt_execute return %d:%s, try:%d", info->id, code, taos_stmt_errstr(stmt), try);
     }
     tscDebug("SML:0x%"PRIx64" taos_stmt_execute inserted %d rows", info->id, taos_stmt_affected_rows(stmt));
-    
+
```

Comment:
如果要重试，当前stmt是不建议继续使用的

Reply:
during the development of this feature, mostly in july. when you fixed a bug about network reconfiguration, the bug fix requires reusing this stmt so that it can work correctly.

Reply:
？